src/ExadelMentorship.BackgroundApp/Program.cs

```diff
@@ -1,13 +1,22 @@
 using ExadelMentorship.BackgroundApp;
 using ExadelMentorship.BusinessLogic.Interfaces.MessageBus;
 using ExadelMentorship.BusinessLogic.Models;
-using ExadelMentorship.BusinessLogic.Services;
+using ExadelMentorship.BusinessLogic.Services.MBus;
+using Microsoft.Extensions.Options;
+using RabbitMQ.Client;
 
 IHost host = Host.CreateDefaultBuilder(args)
     .ConfigureServices(services =>
     {
         services.AddHostedService<MainProcess>();
         services.AddOptions<RabbitMQSettings>().BindConfiguration(nameof(RabbitMQSettings));
+        services.AddSingleton<IConnection>(sp =>
```

Comment:
Still try to make it expression, not statement.
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/lambda-expressions
Lambda expression can be a short form of lambda statement in certain cases. In your case, I don't see any obstacles to make it expression. If it doesn't work, try find out why